src/Sushi.Mediakiwi/Controllers/ContentController.cs

```diff
@@ -628,7 +633,7 @@ private async Task<(ContentItem contentItem, bool isFilled)> getContentItemFromF
                             {
                                 isFilled = true;
                                 var dateTimeValue = new DateTime(unitTime);
-                                System.Globalization.CultureInfo dateCulture = new System.Globalization.CultureInfo("nl-NL");
+                                System.Globalization.CultureInfo dateCulture = new System.Globalization.CultureInfo(culture);
                                 content.Text = dateTimeValue.ToString("dd-MM-yyyy", dateCulture);

```

Comment:
In this case we still get 'dd-mm-yyyy' for en-us
should be:
```suggestion
content.Text = dateTimeValue.ToString(dateCulture.DateTimeFormat.ShortDatePattern);
```